### DISCUSSION TITLE
John Deer Cotton Harvester Blocked by bale

### DISCUSSION THREAD

**Discussion by flipwalker42** [Courseplay/Courseplay_FS22#3355] [Q&A]
Is there a way to prevent the issue of the CP690 harvester canceling the job because it's 'blocked' by the bale it's unloading? I saw a closed recommendation about it, but I didn't see an actual answer to how to prevent it.

**Comment by Tensuko** [comment]
What CP Version?

**Comment by flipwalker42** [comment]
7.4.0.0 according the xml desc. I believe it was current from the ingame mod library. I just doublechecked the github releases and I don't see anything in the release notes up to current that seems to address this. Entirely possible I missed it though.

**Reply by Tensuko** [reply]
Try it anyway!

**Reply by flipwalker42** [reply]
So I just did a test world and even with current dev version from github the worker instantly quits and throws a 'blocked by object' error when unloading a bale. Is there anything else I can provide that might help find the issue?

**Reply by Tensuko** [reply]
Hm I need to check that again.I am sure it did not happen, but will see if it is broken.
But cant tell you when... Busy with other stuff.

**Reply by flipwalker42** [reply]
No worries, thanks for all your work on this mod, it's great!
